### DISCUSSION TITLE
"Select all" should select all the notes in the notes list

### DISCUSSION THREAD

**Discussion by yunesj** [standardnotes/forum#2603] [Suggestions]
In the notes pane, "select all" should select all the notes. Instead, it selects all of the text throughout the app.

**Comment by atmoio** [comment]
Where do you see a select all option?

**Comment by yunesj** [comment]
On mac, via the menu bar ("Edit->Select All"), or via the keyboard shortcut Cmd-a.

**Comment by yunesj** [comment]
The effect is worse without the Folders extension disabled. In this case, it selects all text in the app. With Folders extension enabled, only all of the text of the currently selected note is selected. The standard, expected behavior would be to select multiple notes, hopefully to be able to move them to a folder.

In screenshots, here's what happens when I click a note and Select All:

<img width="729" alt="Screen Shot 2019-11-14 at 2 11 36 PM" src="https://user-images.githubusercontent.com/152988/68900476-c70e3d80-06e8-11ea-81f9-acb62dca7225.png">


The Apple Notes app gives an example of expected behavior. First, I select a note:

<img width="713" alt="Screen Shot 2019-11-14 at 2 07 07 PM" src="https://user-images.githubusercontent.com/152988/68900197-2a4ba000-06e8-11ea-96c2-e03d6c977459.png">

Then, when I Select All, both notes get selected:

<img width="712" alt="Screen Shot 2019-11-14 at 2 07 12 PM" src="https://user-images.githubusercontent.com/152988/68900196-29b30980-06e8-11ea-8b6b-11e442a7b7f7.png">

This allows me to easily delete on a set of notes:

<img width="712" alt="Screen Shot 2019-11-14 at 2 07 18 PM" src="https://user-images.githubusercontent.com/152988/68900195-29b30980-06e8-11ea-87f0-18572019edd4.png">

**Comment by myreli** [comment]
Can replicate. While `cmd+a` now selects all visible notes, through the menu bar (Edit > Select All) it selects the entire text content.
